ucsb-cs156/proj-courses
# pc - replace AdminUpdatesPage placeholder with working AdminUpdatesPage

Deployed at: https://courses-qa.dokku-00.cs.ucsb.edu/

In this PR, we replace the AdminUpdatesPage placeholder with working AdminUpdatesPage.

This page allows an admin to see all of the updates to the MongoDB collection of courses, showing, for each Quarter and Subject Area
* how many new courses, updated courses, and errors occurred
* when the last update took place

The admin can filter the updates by quarter, subjectArea, and can sort them by quarter, subjectArea, or lastUpdate time.  They can also specify the page size.

## Before:

<img width="1173" alt="image" src="https://github.com/user-attachments/assets/28788713-de45-4f5d-9adb-ef760f9b433d" />

## After: 

![adminUpdatesPage](https://github.com/user-attachments/assets/2a8de822-0260-42d7-b389-03cd91d9fc75)

## To Test:

* Login at https://courses-qa.dokku-00.cs.ucsb.edu/
* Navigate to Admin, Updates
* If there are updates records present, you should be able to see them.  If not, try running a job to update the courses database as shown in the animation earlier in this PR description.
* Test each of the controls and make sure that it works properly.
* Test the pagination controls as well to make sure that they work properly.
